pages/api/token/temporary.ts

```diff
@@ -95,14 +95,15 @@ const updateTemporaryToken = async (
 };
 
 const sendTemporaryTokenByEmail = async (email: string, temporaryToken: string) => {
+  const sendTempTokenTemplateID = process.env.TEMPORARY_TOKEN_TEMPLATE_ID;
   const notifyClient = new NotifyClient(
     "https://api.notification.canada.ca",
-    process.env.NOTIFY_API_KEY
+    process.env.NOTIFY_API_KEY ?? "thisIsATestKey"
```

Comment:
I'm curious as to why a default value was added for the 'NOTIFY_API_KEY'.  Would we not want the call to error out if the env var does not exist in the system?  With the default value it might throw a 403 error (forbidden) instead of a conneciton error.  The differece between the two errors could be important when debugging errors in Staging or Production.

Reply:
You're absolutely correct, the default value here won't make it obvious from  the standpoint of an error handling.